### PR TITLE
Better PHPDoc Support for Concrete::getList() and Concrete\Listing::getData()

### PR DESCRIPTION
**Pull request by blankse**

### CODE REVIEW

File: models/DataObject/ClassDefinition.php [hunk 1]
```diff
@@ -408,6 +408,7 @@ public function generateClassFiles($generateDefinitionFile = true)
         $cd .= 'use Pimcore\Model\DataObject\PreGetValueHookInterface;';
         $cd .= "\n\n";
         $cd .= "/**\n";
+        $cd .= '* @method static \\Pimcore\\Model\\DataObject\\'.ucfirst($this->getName()).'\Listing getList()'."\n";
```
Comment (jdreesen) [inline models/DataObject/ClassDefinition.php:411]: Is it `getList()` or `getListing()`?

Comment (blankse) [reply models/DataObject/ClassDefinition.php:411]: getList() https://github.com/pimcore/pimcore/blob/023f10bf34e9c462eeb03ae782740570f7168f01/models/DataObject/AbstractObject.php#L357

**Comment by weisswurstkanone** [comment]
@blankse  Sebastian, could you please merge the master into your branch please? Merci!
Relevant commit is https://github.com/pimcore/pimcore/commit/023f10bf34e9c462eeb03ae782740570f7168f01 and then travis should be happy again.

**Comment by blankse** [comment]
@weisswurstkanone Done

**Comment by brusch** [comment]
Thanks a lot!

**Pull request merged**